src/screens/poa-form/index.js

```diff
@@ -25,30 +25,32 @@ type AddressKeysType =
   | 'region'
   | 'postal_code'
 
-type PoAFormProps = {
+type PoAFormProps = {|
```

Comment:
Looks like this is the Flow way of enforcing an exact type: https://stackoverflow.com/a/48664988/5555637 :heavy_check_mark:

Reply:
correct